shared/components/pages/ActivationPage.jsx

```diff
@@ -236,19 +240,45 @@ export default class ActivationPage extends React.Component {
                                 <span className='ActivationPage__time-to-pass'>
                                     {humanizeDuration(timeToPass, 'second')}
                                 </span>
+
+                                {
+                                    dueTime
+                                    ?   (
+                                        <span className='ActivationPage__expires'>
+                                            <span className='ActivationPage__span-divider'>
+                                                •
+                                            </span>
+                                            <span>{l('expires ')}{getTimeFromNow(dueTime)}</span>
+
+                                        </span>
+                                    )
+                                    : null
+                                }
                             </div>
+
                             <div className='ActivationPage__actions'>
                                 {
-                                    !showUserResult
+                                    canAssigneePass
                                     ? (
-                                        <Button
-                                            ripple
-                                            raised    = {!isSponsored}
-                                            onClick   = {onPass.bind(null, activation)}
-                                            className = 'ActivationPage__btn ActivationPage__pass-btn'
-                                        >
-                                        {l('Pass the test')}
-                                        </Button>
+                                        <div className='ActivationPage__pass-btn-wrapper'>
+                                            <Button
+                                                ripple
+                                                raised    = {!isSponsored}
+                                                onClick   = {onPass.bind(null, activation)}
+                                                className = 'ActivationPage__btn ActivationPage__pass-btn'
+                                            >
+                                            {l('Pass the test')}
+                                            </Button>
+                                            {
+                                                numberOfTriesLeft
+                                                ?
+                                                    <div className='ActivationPage__number-of-tries'>
+                                                        {l('Number of tries left: ')}{numberOfTriesLeft}
```

Comment:
I suppose, it will be better to say

`You have %d tries left `

... or something like that